src/main/java/pages/FeedPage.java

```diff
@@ -1,25 +1,39 @@
 package pages;
 
 import com.codeborne.selenide.Condition;
-import constants.FeedConstants;
+import locators_constants.FeedConstants;
 import java.time.Duration;
 import static com.codeborne.selenide.Selectors.byXpath;
 import static com.codeborne.selenide.Selenide.$;
 
 public class FeedPage implements FeedConstants {
 
-        public pages.FeedPage waitForFeedPageOpened(){
+        public FeedPage waitForFeedPageOpened(){
             $(UPDATE_MOOD_BUTTON_CSS).shouldBe(Condition.visible, Duration.ofSeconds(10));
             return this;
         }
 
-        public RateYourHappiness clickOnUpdateMoodButton(){
+        public RateYourHappinessModal clickOnUpdateMoodButton(){
             $(UPDATE_MOOD_BUTTON_CSS).click();
-            return new RateYourHappiness();
+            return new RateYourHappinessModal();
         }
 
         public String getMoodLevel(){
             return $(byXpath(GET_MOOD_LEVEL_RESULT)).getText();
         }
 
+    public FeedPage clickToHug(){
+        $(byXpath(HUGS_BUTTON)).click();
+        return this;
+    }
+
+    public FeedPage waitForHug(){
+        $(byXpath(HUGS_QUANTITY)).shouldHave(Condition.exactText("1"));
```

Comment:
don't hardcode test values in methods (I mean value "1"). If you want to wait for the specific value, pass this as parameter in method and test.

waitForHugsQuantityDisplayed(String hugsQuantity)

Reply:
done